lib/skills/trolls.js

```diff
@@ -94,33 +109,33 @@ const skill = ({ controller }) => {
     // Check that user has typed three times, or insta-troll
     if (typistEntry.count > 2 || typistEntry.delay === 0) {
       typistEntry.triggered = true;
-      setTimer(bot, message, userID, typistEntry.delay);
+      setTimer(bot, message, user, typistEntry.delay);
 
     // Lower user count if they stop typing before reaching threshold
     } else {
-      resetTimer(bot, message, userID, typistEntry, Math.round(typistEntry.delay / 3));
+      resetTimer(bot, message, user, typistEntry, RESET_DELAY);
     }
   });
 
   // Check if user sent message
-  controller.on('ambient', function(bot, message) {
+  controller.on('ambient', (bot, message) => {
```

Comment:
I didn't realize I had functions here. I copy pasta'd the example from BotKit originally, lol.